ethereumj-core/src/main/java/org/ethereum/vm/VM.java

```diff
@@ -1412,7 +1430,7 @@ public void play(Program program) {
     @Deprecated
     public static void setVmHook(VMHook vmHook) {
         logger.warn("VM.setVmHook(VMHook vmHook) is deprecated method. Define your hook component as a Spring bean.");
```

Comment:
I think we should add line that it will affect only new VM instances

Reply:
What do you mean? Could you explain more detailed?

Reply:
Ahh ok, it looks like a very old code and as it's going to be deprecated, not an isuue